### PR TITLE
fix: Make world safety nag popup title text match the action

### PR DESCRIPTION
**Pull request by budak7273**
Make world safety nag title text match the action being performed instead of always saying 'Copy World' regardless of the button.

Before: (after clicking Datapacks)
image

Before: (after clicking Datapacks)
image

This is my first PR here; pretty sure I followed all of the required process steps. Please let me know if there's anything you need me to change.

### CODE REVIEW

**Comment by budak7273** [comment]
I don't seem to be able to apply labels to this PR, else I'd tag it as `simple change`

File: launcher/ui/pages/instance/WorldListPage.cpp [hunk 1]
```diff
@@ -211,7 +211,7 @@ void WorldListPage::on_actionDatapacks_triggered()
         return;
     }
 
-    if(!worldSafetyNagQuestion())
+    if(!worldSafetyNagQuestion("Open World Datapacks Folder"))
```
Comment (flowln) [inline launcher/ui/pages/instance/WorldListPage.cpp]: For translations to work, I think you need to wrap those strings in `tr()`, the one wrapping `%1` doesn't get those iirc.

File: launcher/ui/pages/instance/WorldListPage.cpp [hunk 2]
```diff
@@ -373,11 +373,11 @@ bool WorldListPage::isWorldSafe(QModelIndex)
     return !m_inst->isRunning();
 }
 
-bool WorldListPage::worldSafetyNagQuestion()
+bool WorldListPage::worldSafetyNagQuestion(const QString &actionType)
 {
     if(!isWorldSafe(getSelectedWorld()))
     {
-        auto result = QMessageBox::question(this, tr("Copy World"), tr("Changing a world while Minecraft is running is potentially unsafe.\nDo you wish to proceed?"));
+        auto result = QMessageBox::question(this, tr("%1").arg(actionType), tr("Changing a world while Minecraft is running is potentially unsafe.\nDo you wish to proceed?"));
```
Comment (flowln) [inline launcher/ui/pages/instance/WorldListPage.cpp]: as per my other comment, this could be just `actionType` instead of using a `tr` i believe

**Review by flowln — COMMENTED** [review]
those `tr()` thingies are hard x.x

**Comment by budak7273** [comment]
Ahh, so that's what `tr`s do ... that makes a lot of sense in retrospect. I couldn't find the function definition, so I assumed it was some sort of special platform-agnostic string maker, and followed the sub-in `arg` examples I found elsewhere.

**Comment by budak7273** [comment]
(I checked to make sure this still works by building again. It still matches my screenshots in the original PR description)

**Review by flowln — APPROVED** [review]
thank you!!

**Review by Scrumplex — APPROVED** [review]

**Review by timoreo22 — APPROVED** [review]

**Pull request merged**